lib/node-token-validation/test/config/OpenIdConfigProvider.spec.ts

```diff
@@ -20,52 +24,63 @@ describe("OpenIdConfigProvider", () => {
 
     describe("constructor", () => {
         it("exports a class", () => {
-            const provider = new OpenIdConfigProvider(config, config.system.networkClient, new Logger(config.system.loggerOptions));
+            const logger = new Logger(config.system.loggerOptions);
```

Comment:
could this be defined at the top of the file instead of defining it in each "it"?

Reply:
Yes, thanks for the catch. Fixed.